CMakeModules/SysioTester.cmake.in

```diff
@@ -65,6 +65,7 @@ find_library(libbscrypto bscrypto @CMAKE_INSTALL_FULL_LIBDIR@ NO_DEFAULT_PATH)
 find_library(libdecrepit decrepit @CMAKE_INSTALL_FULL_LIBDIR@ NO_DEFAULT_PATH)
 find_library(libchainbase chainbase @CMAKE_INSTALL_FULL_LIBDIR@ NO_DEFAULT_PATH)
 find_library(libbuiltins builtins @CMAKE_INSTALL_FULL_LIBDIR@ NO_DEFAULT_PATH)
+find_library(libsodium sodium builtins @CMAKE_INSTALL_FULL_LIBDIR@ NO_DEFAULT_PATH)
```

Comment:
why reference `builtins` in the find_library call? was this a copy-paste typo?  I'd expect just `libsodium sodium @CMAKE_...`  I believe if there's no sodium dir, it'll fallback to assuming that builtins provides the feature (since that's the directory we're already loading for `libbuiltins` above...), which feels like it would just lead to error obfuscation, right?  Anyway, please explain if it was intentional.